src/backoffice/settings/logviewer/workspace/logviewer.context.ts

```diff
@@ -262,10 +254,14 @@ export class UmbLogViewerWorkspaceContext {
 	}
 
 	getLogs = async () => {
-		if (!this.#canShowLogs.getValue()) {
+		if (this.#canShowLogs.getValue() === false) {
```

Comment:
So this works as folowing: 
whenever date range is set we validate the log size. 
if the observable `this.#canShowLogs` is equal to false a [warning is displayed](https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/602a7ea59a7187fa8a5a0735a6dc271af3142e25/src/backoffice/settings/logviewer/workspace/views/overview/log-overview-view.element.ts#L70-L71) instead of saved searches/log messages, so they will not see any logs if that changes value. 
This is added so it just does not make the log request when it cannot show them, I dont know what comes back from the endpoint if therer are to many messages in a given date range, but I guess there is no point in making the request for logs at all if there are to many to show. I'm not sure if I understand your commnet correctly actually.